Packs/Twitter/Integrations/Twitterv2/Twitterv2_test.py

```diff
@@ -0,0 +1,144 @@
+import json
+import io
+import freezegun
+import pytest
+
+
+class ResponseMock:
+    def __init__(self, _json={}):
+        self.status_code = 200
+        self._json = _json
+
+    def json(self):
+        return self._json
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+test_data = util_load_json('test_data/test_data.json')
+
+
+@pytest.mark.parametrize('response, expected_output, expected_human_readable, expected_raw', [
+                         (test_data['search_tweets_response'], test_data['search_tweet_output'],
+                          test_data['search_tweets_human_readable'], test_data['search_tweets_response'])])
+def test_twitter_tweet_search_command(mocker, response, expected_output, expected_human_readable, expected_raw):
+    """
+    Given:
+        - script_results
+    When:
+        - after running a script on the XDRIR integration
+    Then:
+        - choose the last script that is related to the process list.
+        (contains the Name,CPU,Memory in the entry).
+    """
+    from Twitterv2 import Client, twitter_tweet_search_command
+
+    client = Client(base_url='some_base_url', verify=False,
+                    headers={'Authorization': 'Bearer 000'})
+    args = {
```

Comment:
Lets add a test also with start_time and end_time
Also check for arguments, what will return if only one of them is provided, and if the limit is not as expected